app/tutorials/weather-warning/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
do we really need this?

Reply:
I'm mocking the `VehicleParameters`-class which is final and can't be mocked without this. However, maybe there's a better way to write the test 😅

Reply:
Yeah, actually this is only needed because WeatherWarningApp uses ` getOs().requestVehicleParametersUpdate().changeColor(Color.RED).apply();`.. we could just delete this line and set the color in the `mapping_config.json`

Reply:
Keep this, since we also mock `RoutingResponse#getBestRoute()` which is also final